src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs

```diff
@@ -46,7 +46,7 @@ public override async Task<CodeAction> GetFixAsync(FixAllContext fixAllContext)
 
         #endregion
 
-        internal override async Task<CodeAction> GetFixAsync(
+        internal async Task<CodeAction> GetFixAsync(
```

Comment:
❔ Can these be `private` instead?

Reply:
These are used by AbstractSuppressionCodeFixProvider.

Reply:
called from other type.  That said, feels like `public` miht be better here.  @mavasani any preference?